CONTRIBUTING.md

```diff
@@ -373,6 +374,150 @@ regression    =   1.44 sec*proc (1 test)
 Total Test time (real) =   1.47 sec
 ```
 
+#### Adding Performance Regression Tests
+
+Regression tests can also be added for the purposes of testing the performance.
+Adding performance tests does require understanding a bit about how the ci
+infrastructure is setup. Currently, the performance tests are setup to run on
+dedicated Power9 nodes that are provided by LANL, this is to ensure consistency
+in the result. However, because the performance tests are running on LANL
+machines it means that it is not straightforward to make the results viewable
+to non LANL contributors. To make the results of the regression tests viewable
+on github a custom github application was developed to be used with Parthenon
+which we will call the Parthenon Performance Metrics App (PPMA).
+
+The PPMA is series of python scripts located in the parthenon repository in
+./scripts/python/packages/parthenon_performance_app. The PPMA has the following
+responsibilities:
+
+1. Submit a status check associated with the commit being analyzed to the
+   Parthenon repository
+2. Read in performance metrics from regression tests in a build directory
+3. Compare the current metrics with metrics stored in the parthenon.wiki, the
+   files in the wiki should contain performance metrics from previous runs.
+4. Generate images showing the performance numbers.
+5. Add images to an orphan branch in the parthenon repository (branch name is
+   figures).
+6. Add new performance metrics to the appropriate performance .json file stored
+   in the wiki. These json files are labeled with the following format:
+   performance_metrics_current-branch.json, this means each branch should
+   have its own .json performance file.
+7. Create or overwrite a wiki page that contains links to the performance
+   figures located in the parthenon wiki repository.
+8. Upload the wiki page the images and performance metrics to the Parthenon
+   repository. The wiki page will be named with the following format:
+   current-branch_target-branch.md
+   E.g. If you were merging hotfix branch into develop the wiki page would be
+   named: hotfix_develop.md, each pr should have it's own wiki page. For performance
```

Comment:
```suggestion
   named: hotfix_develop.md, each PR should have it's own Wiki page. For performance
```
What if we have more than one PR with the same branch names? I.e. PR X merges "hotfix" into "develop" and then later PR Y also merges "hotfix" into develop? Will the second overwrite the first? Would it be possible to add the PR number to the Wiki page name?